src/ElectricPokemon.java

```diff
@@ -0,0 +1,95 @@
+import java.util.Arrays;
+import java.util.List;
+
+public class ElectricPokemon extends Pokemon {
+
+    private static final String type = "electric";
+    List<String> attacks = Arrays.asList("ThunderPunch", "ElectroBall", "Thunder", "VoltTackle");
+
+    public ElectricPokemon(String name, int level, int hp, String food, String sound) {
+        super(name, level, hp, food, sound, type);
+    }
+
+    void thunderPunch(Pokemon name, Pokemon enemy) {
```

Comment:
Je kiest hier voor "package private" en niet voor "public" als acces modifier. Voor nu is dat gen probleem, maar stel je zet al je pokemons in een "pokemons" mapje, dan kun je deze methodes niet meer buiten dat mapje gebruiken. Dus ook niet in de PokemonGymImpl die dan buiten dat mapje staat.

Reply:
Je hebt de aanval methodes goed uitgewerkt.